pages/glossary/non-empty.md

```diff
@@ -3,15 +3,4 @@ title: Non-empty text string
 key: non-empty
```

Comment:
You can't just change this definition. Changing this changes the meaning of a whole bunch of existing rules. If you want to change the meaning of this definition you'll need to create a different definition for the rules that currently use this. Please do that in a separate PR so that we can have a record of it.